src/core/lib/iomgr/wakeup_fd_pipe.cc

```diff
@@ -71,8 +71,9 @@ static grpc_error* pipe_consume(grpc_wakeup_fd* fd_info) {
 
 static grpc_error* pipe_wakeup(grpc_wakeup_fd* fd_info) {
   char c = 0;
-  while (write(fd_info->write_fd, &c, 1) != 1 && errno == EINTR)
+  while (write(fd_info->write_fd, &c, 1) != 1 && errno == EINTR) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done